### ISSUE TITLE
Symbolic link to data directory from bin directory no longer works

### ISSUE THREAD

**Issue by stephengtuggy** [vegastrike/Vega-Strike-Engine-Source#316]
I was using a little trick to load @evertvorster 's galaxy backgrounds without having to manually specify the path to them as a command-line argument to vegastrike. My trick was to place a symbolic link in the bin directory, named data, linking to the Assets-Production directory. As of #298 , that no longer works. I'm still not sure why #298 broke it, but it did.

I don't see this as a high-priority item. It's probably fine to go in the backlog.

**Comment by BenjamenMeyer** [comment]
@stephengtuggy is this still an issue?

**Comment by stephengtuggy** [comment]
@BenjamenMeyer Yes, I'm pretty sure it is. What I believe would fix it is to get the canonical path of the binary, not just its relative path.

**Comment by stephengtuggy** [comment]
@BenjamenMeyer this issue can probably be closed. I think I fixed it as of 0.8.x. At a minimum, it has become less relevant, with the switch to the `vegastrike-engine` scheme of things.
